src/valora-dapp-list.json

```diff
@@ -269,15 +269,6 @@
       "listOnAndroid": true,
       "listOnIos": true
     },
-    {
-      "name": "Pixpress",
-      "id": "pixpress",
-      "categoryId": "exchanges",
-      "categories": ["exchanges", "earn"],
-      "url": "https://pixpress.pixelava.space",
-      "listOnAndroid": true,
-      "listOnIos": true
```

Comment:
Should we turn these to `false` instead of deleting for now?
like we did for https://github.com/valora-inc/dapp-list/pull/469

Reply:
It should be fine to delete, if we want to we can re-add it later.

Reply:
I think given that's not a very active dapp and it's still on V1 it'll be fine to remove. If desired by the author we can accept a new submission when v2 is supported.